lib/mockbuilder.py

```diff
@@ -128,9 +128,7 @@ def _install_external_dependencies(self, package):
             cmd = self.common_mock_args
             install = " --install"
             for dep in package.build_dependencies:
-                dep_rpms_glob = os.path.join(dep.build_results_dir, "*.rpm")
-                rpm_files = glob.iglob(dep_rpms_glob)
-                install = " ".join([install, " ".join(rpm_files)])
+                install = " ".join([install, " ".join(dep.cached_build_results)])
```

Comment:
The point is that "results" are a generic term, it could be logs, built packages or extra outputs (eg a yum repository config file). 

You may use cached_built_files or cached_built_files_paths.